CGCharter-1727386911.html

```diff
@@ -229,7 +229,7 @@ <h2 id="decision">
     <p>
       This group will seek to make decisions where there is consensus. The Group makes decisions in the following ways: either Committers assess
       consensus, or the Chair assesses consensus, or where consensus isn't
```

Comment:
This change (lines 230-231) goes along with another (lines 232-233)
```suggestion
      This group will seek to make decisions where there is consensus. The
      Group makes decisions in the following ways: either Committers assess
      consensus; the Chair assesses consensus; or, where consensus isn't
```